src/main/webapp/content/scss/partials/_report-management.scss

```diff
@@ -45,6 +45,7 @@
     width: 19px;
     height: 19px;
     margin: 5px;
+    fill: #676a6c;
```

Comment:
we are using this color any more...just confirm with venkat...also any color should be deifined in_variable.scss file